### ISSUE TITLE
ERROR: Programmatically selecting a PushRow/custom ImageRow

### ISSUE THREAD

**Issue by pjr0001** [xmartlabs/Eureka#859]
I'm using push rows & image picker row example in one of my forms. I'd like to select one of these rows (either a pushrow or the custom image picker row) automatically when the form loads, so that the user is prompted to pick a picture, and then continue filling out information about that photo.

I've tried the following advice after searching

```row.didSelect() ```

```  
 if let row = form.rowBy(tag: "Title") {
     let indexPath = IndexPath(item: 0, section: 0)
     tableView?.selectRow(at: indexPath, animated: true, scrollPosition: UITableViewScrollPosition.none)
     tableView?.delegate?.tableView?(tableView!, didSelectRowAt: indexPath)
}
```
both of this cause an error while trying to unwrap an optional.
``` fatal error: unexpectedly found nil while unwrapping an Optional value ```

the error arises in FormViewController customDidSelect()
``` presentationMode.present(controller, row: self, presentingController: self.cell.formViewController()!) ```

**tl;dr** - how do programmatically simulate the "tap" gesture of a push row on a form, causing it to open?

**Comment by santiagofm** [comment]
Hey @pjr0001 !
I tried your solution and it worked fine with the row with tag `Repeat` in the `NativeEventFormViewController` of the Example project:
```swift
if let row = form.rowBy(tag: "Repeat") as? PushRow<RepeatInterval>, let indexPath = row.indexPath {
    tableView?.selectRow(at: indexPath, animated: true, scrollPosition:.none)
    tableView?.delegate?.tableView?(tableView!, didSelectRowAt: indexPath)
}
```

Cheers

**Comment by palla89** [comment]
Hi! I have the same problem as pjr0001, with same error. 
I tried your code @santiagofm but it gives me the same exact crash on presentationMode.present(controller, row: self, presentingController: self.cell.formViewController()!)

Any hint on what it could be?
